keel-ec2-plugin/src/main/kotlin/com/netflix/spinnaker/keel/ec2/jackson/KeelEc2ApiModule.kt

```diff
@@ -0,0 +1,96 @@
+
+import com.fasterxml.jackson.databind.BeanDescription
+import com.fasterxml.jackson.databind.DeserializationConfig
+import com.fasterxml.jackson.databind.JavaType
+import com.fasterxml.jackson.databind.JsonDeserializer
+import com.fasterxml.jackson.databind.JsonSerializer
+import com.fasterxml.jackson.databind.Module
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.databind.SerializationConfig
+import com.fasterxml.jackson.databind.deser.Deserializers
+import com.fasterxml.jackson.databind.module.SimpleModule
+import com.fasterxml.jackson.databind.ser.Serializers
+import com.netflix.spinnaker.keel.api.ec2.ApplicationLoadBalancerSpec
+import com.netflix.spinnaker.keel.api.ec2.ArtifactImageProvider
+import com.netflix.spinnaker.keel.api.ec2.ClassicLoadBalancerSpec
+import com.netflix.spinnaker.keel.api.ec2.ClusterDependencies
+import com.netflix.spinnaker.keel.api.ec2.ClusterSpec
+import com.netflix.spinnaker.keel.api.ec2.ClusterSpec.HealthSpec
+import com.netflix.spinnaker.keel.api.ec2.ClusterSpec.ServerGroupSpec
+import com.netflix.spinnaker.keel.api.ec2.CustomizedMetricSpecification
+import com.netflix.spinnaker.keel.api.ec2.Health
+import com.netflix.spinnaker.keel.api.ec2.ImageProvider
+import com.netflix.spinnaker.keel.api.ec2.IngressPorts
+import com.netflix.spinnaker.keel.api.ec2.Scaling
+import com.netflix.spinnaker.keel.api.ec2.SecurityGroupRule
+import com.netflix.spinnaker.keel.api.ec2.SecurityGroupSpec
+import com.netflix.spinnaker.keel.api.ec2.StepAdjustment
+import com.netflix.spinnaker.keel.api.ec2.StepScalingPolicy
+import com.netflix.spinnaker.keel.api.ec2.TargetTrackingPolicy
+import com.netflix.spinnaker.keel.ec2.jackson.mixins.ApplicationLoadBalancerSpecMixin
+import com.netflix.spinnaker.keel.ec2.jackson.mixins.ArtifactImageProviderMixin
+import com.netflix.spinnaker.keel.ec2.jackson.mixins.ClassicLoadBalancerSpecMixin
+import com.netflix.spinnaker.keel.ec2.jackson.mixins.ClusterDependenciesMixin
+import com.netflix.spinnaker.keel.ec2.jackson.mixins.ClusterSpecMixin
+import com.netflix.spinnaker.keel.ec2.jackson.mixins.CustomizedMetricSpecificationMixin
+import com.netflix.spinnaker.keel.ec2.jackson.mixins.HealthMixin
+import com.netflix.spinnaker.keel.ec2.jackson.mixins.HealthSpecMixin
+import com.netflix.spinnaker.keel.ec2.jackson.mixins.ScalingMixin
+import com.netflix.spinnaker.keel.ec2.jackson.mixins.SecurityGroupRuleMixin
+import com.netflix.spinnaker.keel.ec2.jackson.mixins.SecurityGroupSpecMixin
+import com.netflix.spinnaker.keel.ec2.jackson.mixins.ServerGroupSpecMixin
+import com.netflix.spinnaker.keel.ec2.jackson.mixins.StepAdjustmentMixin
+import com.netflix.spinnaker.keel.ec2.jackson.mixins.StepScalingPolicyMixin
+import com.netflix.spinnaker.keel.ec2.jackson.mixins.TargetTrackingPolicyMixin
+
+fun ObjectMapper.registerKeelEc2ApiModule(): ObjectMapper = registerModule(KeelEc2ApiModule)
+
+object KeelEc2ApiModule : SimpleModule("Keel EC2 API") {
```

Comment:
All the custom Jackson stuff for classes in `keel-ec2-api` that was previously done with annotations